src/select/stateful-select-container.js

```diff
@@ -41,6 +42,15 @@ class StatefulSelectContainer extends React.Component<
     return onChange(e, params);
   };
 
+  onTextInputChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    this.stateReducer(STATE_CHANGE_TYPE.textChange, e, {
+      type: STATE_CHANGE_TYPE.textChange,
+      textValue: e.target.value,
+    });
+    const {onTextInputChange} = this.props;
+    return onTextInputChange(e);
+  };
```

Comment:
Can we move the `textValue` state into the stateless `Select` component? As a customer I would want to use a stateless Select but I don't want to manage the `textValue` myself.

cc @abmai @nadiia thoughts?

Reply:
textValue is in the stateless `Select`
![image](https://user-images.githubusercontent.com/1851469/46307180-120fd480-c56b-11e8-9514-92a3866d2463.png)